pkg/cmd/build/build_test.go

```diff
@@ -236,3 +240,127 @@ func Test_OptsFromBuildInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestOptsFromBuildInfo(t *testing.T) {
```

Comment:
I think the tests name doesn't match the function we actually test. It seems this is testing `extractFromContextAndDockerfile`

Reply:
Right